nss-day-cohort-47/ldcc-bvinson23
# Bv toppingdropdown

# Description
Added a filter to the drop down using an event listener to show only the snacks with that topping
Fixes # (issue)
## Type of change
# Testing Instructions
Serve the site, launch the json-server, select a topping from the drop down. You should see the corresponding snacks on the DOM
# Checklist:
